setup.cfg

```diff
@@ -70,6 +70,10 @@ snowflake =
 apache.livy =
     apache-airflow-providers-apache-livy
     paramiko
+apache.hive =
+    apache-airflow-providers-apache-hive
+    sasl>=0.3.1; python_version>="3.9"
```

Comment:
does this means we will have a problem in running this sensor below python3.9 env?

Reply:
had added for the sasl issue while building ci image we don't need it removed that from dev Dockerfile